includes/drush/custom/custom.drush.inc

```diff
@@ -20,6 +20,15 @@ function custom_drush_command() {
     'aliases' => ['fr-site', 'frsi'],
   ];
 
+  $items['check-unused-modules'] = [
```

Comment:
Should we align and namespace all related drush commands introduced in 3.0.22?
```
- check-unused-modules
- platform-overridden-modules
- toolkit-contrib-authorised-security
- toolkit-contrib-update
```
@zarabatana , @verbruggenalex ?

Reply:
My proposal:
- toolkit-check-modules-unused
- toolkit-check-modules-platform
- toolkit-check-modules-authorized
- toolkit-check-modules-updated

Reply:
I would say
```
toolkit-check-modules-unused
toolkit-check-modules-restricted
toolkit-check-modules-authorized
toolkit-check-modules-security-advisory
```

Reply:
Agreed... I'll create a ticket to align the names.